### ISSUE TITLE
Parser fails to handle multi-line copyright input

### ISSUE THREAD

**Issue by kellnerd** [kellnerd/musicbrainz-bookmarklets#16]
Just now when I tried the following, only the phonographic copyright was added:
© «2017 The Media Champ»
℗ «2003 The Media Champ»

It was working before, but I think it's because of the cache. I had another release that had the same label, but only a phonographic copyright line and I had to do a search. When I entered this, it didn't do a search and just added this as a phonographic copyright and ignored the copyright line entirely. I like the fact that it just adds the label once you've found it before, but obviously, this wasn't intended.

_Originally posted by @tigerman325 in https://github.com/ROpdebee/mb-userscripts/issues/77#issuecomment-1006981004_

**Comment by kellnerd** [comment]
I assume you have pasted both lines into the prompt popup at the same time?

I just tried this in Firefox and it suggested me to add a (C) relationship for 'The Media Champ ℗ 2003 The Media Champ'. Apparently Firefox implementation of the prompt implicitly converts the line break into a space which breaks the end-of-name detection.
And your browser (Chrome?) probably respected the line break, but my pattern matched only the second line because I did not correctly set up multi-line mode (in which line breaks and the end of the text are treated identically) and the first occurrence of the label name is not terminated by a comma or a full stop either.

Fixed in 225500396113b73038dfe075d375f7c4610cdd61, I will publish a new release as soon as I have replaced the prompt by a proper UI with a multi-line textarea input (which works as expected in all browsers). I will also work on the option to add recording-level relationships while am I already working on the UI.
